ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/shared-phz/provision/cucushift-installer-rehearse-aws-ipi-shared-phz-provision-chain.yaml

```diff
@@ -0,0 +1,36 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-shared-phz-provision
```

Comment:
Compared this chain with `cucushift-installer-rehearse-aws-ipi-shared-vpc-phz-provision`, seem like the only difference is the default value of `ENABLE_SHARED_VPC`, if that, I think we can directly call `cucushift-installer-rehearse-aws-ipi-shared-vpc-phz-provision` with a different default value in install workflow.

Reply:
Updated